tsfile/src/main/java/org/apache/iotdb/tsfile/write/writer/NativeRestorableIOWriter.java

```diff
@@ -53,6 +53,15 @@ long getTruncatedPosition() {
   }
 
   public NativeRestorableIOWriter(File file) throws IOException {
+    this(file, true);
```

Comment:
Perhaps we should set the default to false to keep the behavior "as is".
But otherwise "true" is more comfortable for the user.
I agree with both, just wanted to bring this point up.

Reply:
Hi, I think the default value is true is better, because the target of using this class is for recovering broken files...

For a reader class (not implemented now), the default value of `autoRepair` should be false.

Reply:
agree!